client/src/components/ChatListScreen/ChatList.test.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import { ApolloProvider } from 'react-apollo-hooks';
 import ReactDOM from 'react-dom';
 import { cleanup, render, fireEvent, wait, waitForDomChange } from '@testing-library/react';
 import { createBrowserHistory } from 'history';
-import ChatsList from './ChatList';
+import { mockApolloClient } from '../../test-helpers';
+import ChatList, { getChatsQuery } from './ChatList';
```

Comment:
hey @milenazuccarelli 
I think the import from ChatList should be
```
import ChatsList, { getChatsQuery } from './ChatList';
```

I might be wrong but I think it will fix the CircleCI fail.